content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -351,6 +351,12 @@ You now need to ensure that the kubectl completion script gets sourced in all yo
     ```shell
     kubectl completion bash >/etc/bash_completion.d/kubectl
     ```
+- Should you have an alias for kubectl, completion can can be extended to work for it as well:
```

Comment:
pls remove dup 'can'.

Reply:
How about:
```suggestion
- If you have an alias for kubectl, you can extend shell completion to work with that alias:
``

Reply:
ha been updated as suggested, in all locations.